src/sections/Home/So-Special-Section/so-special-style.js

```diff
@@ -7,6 +7,8 @@ const SoSpecialWrapper = styled.div`
     align-items:center;
     padding:0rem 3rem 2rem 3rem;
     overflow:hidden;
+    font-family:"Qanelas Soft";
```

Comment:
Fonts shouldn't be used this way. Qanelas soft is specifically for headings, if you want to use this font use the `<h*>` tags